src/store/conversationsStore.js

```diff
@@ -100,6 +100,10 @@ const mutations = {
 	purgeConversationsStore(state) {
 		Object.assign(state, getDefaultState())
 	},
+
+	setConversationDescription(state, { token, description }) {
+		Vue.set(state.conversations[token], 'description', description)
```

Comment:
nice, didn't know there was a way that wouldn't require re-adding the conversation object as it's done in other places :+1:

Reply:
yeah I was surprised to see that In other places it was replacing everything. It's actually just a key that needs to be modified